Extension/src/Debugger/configurationProvider.ts

```diff
@@ -427,13 +472,17 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
                 const newConfig: CppDebugConfiguration = { ...defaultTemplateConfig }; // Copy enumerables and properties
                 newConfig.existing = false;
 
-                newConfig.name = configPrefix + compilerName + " " + this.buildAndDebugActiveFileStr();
+                newConfig.name = `${configPrefix}${compilerName} [Debugger: ${newConfig.type}] ${this.buildAndDebugActiveFileStr()}`;
                 newConfig.preLaunchTask = task.name;
-                if (newConfig.type === DebuggerType.cppdbg) {
-                    newConfig.externalConsole = false;
-                } else {
-                    newConfig.console = "integratedTerminal";
+                switch (newConfig.type) {
+                    case DebuggerType.cppdbg:
+                        newConfig.externalConsole = false;
+                        break;
+                    case DebuggerType.cppvsdbg:
+                        newConfig.externalConsole = false;
```

Comment:
Could you help me understand why cppvsdbg is changing here?  I was under the impression that `externalConsole` was the old, deprecated property and `console` was the newer, preferred one to set.

Reply:
That is a really good question.

I .... remember running into a problem where I got an error, and now that I think about it, I can't remember why it moved me to fix it here with that. I may have made a mistake and got this backwards. lemme look into it

Reply:
Ah, I see what happened. I fixed this coming from two directions and when coming back to it made the error when I converted what I was doing to a case. Weird. Fixing....